### ISSUE TITLE
Components doesn't have accessibility

### ISSUE THREAD

**Issue by mohammedzamakhan** [WiziShop/ng-wizi-bulma#10]
I tried the modal, dropdown, edit in place, tooltip and didn't find them to have accessibility for keyboard users

**Comment by JumBay** [comment]
You're right, we didn't focus on this. Feel free to provide us PR. Thanks
